src/app/ubs-admin/components/ubs-admin-order-client-info/ubs-admin-order-client-info.component.html

```diff
@@ -0,0 +1,69 @@
+<div class="container">
+  <form class="form" [formGroup]="clientInfoForm">
+    <div>
+      <h3 class="form__headline">Інформація про клієнта</h3>
+    </div>
+    <div class="form__wrapper row d-flex">
+      <div class="form__block col-md-4 col-sm-6">
+        <div class="form-group">
+          <p class="form__subtitle">Клієнт</p>
+          <label class="form__label" for="client-name">Ім'я</label>
+          <input class="form-control" type="text" id="client-name" placeholder="Placeholder"
+          formControlName="clientName"
+          />
+        </div>
+        <div class="form-group">
+          <label class="form__label" for="client-tel">Телефон</label>
+          <input class="form-control" type="tel" id="client-tel" placeholder="Placeholder" 
+          formControlName="clientPhone"
+          />
+        </div>
+        <div class="form-group">
+          <label class="form__label" for="client-email">Email</label>
+          <input class="form-control" type="email" id="client-email" placeholder="Placeholder" 
+          formControlName="clientEmail"
+          />
+        </div>
+      </div>
+
+      <div class="form__block col-md-4 col-sm-6 offset-md-1">
+        <div class="form-group">
+          <p class="form__subtitle">Отримувач</p>
+          <label class="form__label" for="recipient-name">Ім'я</label>
+          <input class="form-control" type="text" id="recipient-name" placeholder="Placeholder" 
+          formControlName="recipientName"
+          />
+        </div>
+        <div class="form-group">
+          <label class="form__label" for="recipient-tel">Телефон</label>
+          <input class="form-control" type="tel" id="recipient-tel" placeholder="Placeholder" 
+          formControlName="recipientPhone"
+          />
+        </div>
+        <div class="form-group">
+          <label class="form__label" for="recipient-email">Email</label>
+          <input class="form-control" type="email" id="recipient-email" placeholder="Placeholder" 
+          formControlName="recipientEmail"
+          />
+        </div>
+      </div>
+    </div>
+
+    <div class="violation">
+      <p class="form__subtitle">Порушення клієнта</p>
+      <div class="violation__block d-flex">
+        <div class="violation__item">
+          <small class="form__label">Всього</small>
+          <p>{{ violations }}</p>
+        </div>
+        <div class="violation__item">
+          <small class="form__label">Поточне замовлення</small>
+          <p>{{ currentViolationAmount }}</p>
+        </div>
+      </div>
+
+      <button class="btn" disabled="violations > 1" type="button">Додати порушення</button>
```

Comment:
This button will always be disabled because it has `disabled` attribute
You should use it as `[attr.disabled]="violations > 1"`

Reply:
Fixed